docs/application/tizen-studio-0/setup/tau_quick_start_guide.md

```diff
@@ -0,0 +1,37 @@
+# How to Develop Wearable Web Application with TAU
```

Comment:
Why quick guides are here?
Move to the right place.

Reply:
All these files are in-progress; I will make another landing page for these files or will put in their appropriate places in future.